Fortune.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+"""Fortune wrapper extension"""
+
+from albertv0 import *
+from shutil import which
+import subprocess as sp
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "Fortune"
+__version__ = "1.0"
+__trigger__ = "fortune"
+__author__ = "Kelvin Wong"
+__dependencies__ = ["fortune"]
+
+if which("fortune") is None:
+    raise("'fortune' is not in $PATH.")
+
+iconPath = iconLookup("font")
+
+
+def handleQuery(query):
+    if query.isTriggered:
+        return generateFortune(query)
+
+
+def generateFortune(query):
+    item = Item(
+        id=__prettyname__,
+        icon=iconPath,
+        text=__prettyname__,
+        completion=query.rawString
+    )
+    try:
+        item.text = sp.check_output(['fortune', '-s'],
+                                    stderr=sp.STDOUT).decode().strip()
```

Comment:
Why `stderr=sp.STDOUT`?

Reply:
Indeed it should be not required. Was fiddling around the error handling but left this line there.